app/assets/javascripts/views/postChatSurvey/PostChatSurveyWebchat.js

```diff
@@ -1,8 +1,6 @@
 const html = `
     <div id="postChatSurvey">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l" id="legend_give_feedback" tabindex="-1">
-          <h1 class="govuk-fieldset__heading">Give feedback</h1>
-        </legend>
+          <h2 class="govuk-fieldset__heading" id="legend_give_feedback" tabindex="-1">Give feedback</h2>
```

Comment:
Again, can we remove the class as it's not in a fieldset?

Reply:
> Again, can we remove the class as it's not in a fieldset?

Done